services/IoT/applications/libraries/nodejs.md

```diff
@@ -179,12 +179,12 @@ When the application client encounters an error, an *error* event is generated.
 
 Events are the mechanism by which devices publish data to the {{site.data.keyword.iot_short_notm}} instance. The device controls the content of the event and assigns a name for each event that it sends.
 
-When an event is received by the {{site.data.keyword.iot_short_notm}} instance, the credentials of the received event identify the sending device, making it impossible for a device to impersonate another device.
+When an event is received by the {{site.data.keyword.iot_short_notm}} instance, the credentials of the received event identify the sending device, which makes it impossible for a device to impersonate another device.
 
 
 
 
-By default, applications subscribe to all events from all connected devices. Use the type, ID, event, and msgFormat parameters to control the scope of the subscription. A single client can support multiple subscriptions. The following code samples provide examples of how to subscribe to devices that are dependent on device type, ID, event, and msgFormat parameters.
+By default, applications subscribe to all events from all connected devices. Use the type, ID, event, and msgFormat parameters to control the scope of the subscription. A single client can support multiple subscriptions. The following code samples provide examples of how to subscribe to devices that are dependent on device type, ID, event, and msgFormat parameters:
```

Comment:
"The following code samples provide examples of how to subscribe to devices that are dependent on device type, ID, event, and msgFormat parameters:"
the phrase "that are dependent on device type, ID, event, and msgFormat parameters" is kind of dangling at the end, which makes it hard to understand whether the examples are dependent on the parameters or the devices are dependent on the paramters. Needs to be clarified/rewritten.